lib/webpacker/commands.rb

```diff
@@ -5,12 +5,34 @@ def initialize(webpacker)
     @webpacker = webpacker
   end
 
-  def clean(count = 2)
-    if config.public_output_path.exist? && config.public_manifest_path.exist? && versions.count > count
-      versions.drop(count).flat_map(&:last).each do |file|
-        File.delete(file) if File.file?(file)
-        logger.info "Removed #{file}"
-      end
+  # Cleanup old assets in the compile directory. By default it will
+  # keep the latest version, 2 backups created within the past hour.
+  #
+  # Examples
+  #
+  #   To force only 1 backup to be kept, set count=1 and age=0.
+  #
+  #   To only keep files created within the last 10 minutes, set count=0 and
+  #   age=600.
```

Comment:
@gauravtiwari if you really want to clean the directory, saving what Webpack created, why not just skip the whole `clean` and instead clear the destination directory before the build starts?

Reply:
I was investigating this code today and I think those comments are misleading (or I'm mixing booleans in my head).

if we pass `age` as 0, then `max_age` will always be bigger, so the whole `max_age < age && index < count` will be false always and then `everything` will be removed independently of the value of `index`. 
In this case, `age` should be something like `Float::INFINITY`

the same story for the second line, you can't pass `count` as 0, should be infinity too